spec/models/person_spec.rb

```diff
@@ -29,11 +29,11 @@
       end
 
       it 'selects distinct people' do
-        aspect = bob.aspects.create(:name => 'hilarious people')
+        aspect = bob.aspects.create(name: 'hilarious people')
         aspect.contacts << bob.contact_for(eve.person)
-        person_ids = Person.for_json.joins(:contacts => :aspect_memberships).
-          where(:contacts => {:user_id => bob.id},
-               :aspect_memberships => {:aspect_id => bob.aspect_ids}).map{|p| p.id}
+        person_ids = Person.for_json.joins(contacts: :aspect_memberships).
```

Comment:
Place the . on the next line, together with the method name.<br>Pass `&:id` as an argument to `map` instead of a block.